covid19-etl/utils/metadata_helper.py

```diff
@@ -119,39 +93,24 @@ def get_existing_data_jhu(self):
 
     def get_latest_submitted_date_idph(self):
         """
-        Queries Peregrine for the existing `summary_clinical` data.
-
-        { summary_clinical (first: 1, project_id: <...>) { date } }
-
+        Queries Guppy for the existing `location` data.
         Returns the latest submitted date as Python "datetime.date"
         """
-        print("Getting latest date from Peregrine...")
-        query_string = (
-            '{ summary_clinical (first: 1, order_by_desc: "date", project_id: "'
-            + self.project_id
-            + '") { submitter_id date } }'
-        )
-        response = requests.post(
-            "{}/api/v0/submission/graphql".format(self.base_url),
-            json={"query": query_string, "variables": None},
-            headers=self.headers,
-        )
-        assert (
-            response.status_code == 200
-        ), "Unable to query Peregrine for existing data: {}\n{}".format(
-            response.status_code, response.text
-        )
-        try:
-            query_res = json.loads(response.text)
-        except:
-            print(f"Peregrine did not return JSON: {response.text}")
-            raise
-
-        if len(query_res["data"]["summary_clinical"]) < 1:
-            return None
-
-        sc = query_res["data"]["summary_clinical"][0]
-        latest_submitted_date = datetime.datetime.strptime(sc["date"], "%Y-%m-%d")
+        print("Getting latest date from Guppy...")
```

Comment:
Typos

Reply:
What typo?

Reply:
should be `getting the latest date from Guppy`